addons/block_code/simple_nodes/simple_character/simple_character.gd

```diff
@@ -83,24 +104,44 @@ func _ready():
 
 func simple_setup():
 	add_to_group("affected_by_gravity", true)
+	_setup_actions()
 	_texture_updated()
 
 
-func get_custom_class():
-	return "SimpleCharacter"
+func _setup_actions():
+	if(InputMap.has_action("player_1_left")):
+		return
+	
+	InputMap.add_action("player_1_left")
+	InputMap.add_action("player_1_right")
+	InputMap.add_action("player_1_up")
+	InputMap.add_action("player_1_down")
+	InputMap.add_action("player_2_left")
+	InputMap.add_action("player_2_right")
+	InputMap.add_action("player_2_up")
+	InputMap.add_action("player_2_down")
+
+	for player in PLAYER_KEYS:
+		for action in PLAYER_KEYS[player]:
+			var e = InputEventKey.new()
+			e.keycode = PLAYER_KEYS[player][action]
```

Comment:
Previously the input vector was calculated with `Input.is_physical_key_pressed` which corresponds to `InputEventKey.physical_keycode`. See commit 16ab468742f47901174e43ab19608380ba6d8c52 for why this is significant. Please adjust this to use `physical_keycode` instead.

Reply:
good catch, I fixed it as part of 8be0a70